docker-compose.yml

```diff
@@ -569,38 +573,37 @@ services:
     depends_on:
       - db
 
-  koku-minio:
-    image: local-minio:latest
-    build:
-        context: .
-        dockerfile: Dockerfile.minio
-        args:
-          USER_ID: ${USER_ID-1000}
-    container_name: kokuminio
+  minio:
+    container_name: koku-minio
+    image: minio/minio:RELEASE.2022-05-26T05-48-41Z.hotfix.15f13935a
```

Comment:
I know that this is the image we were using previously, but maybe we should update this to a newer image? It's definitely not important, but a nice to have since we're already changing these files around.

Reply:
I'm totally ok with updating the image. Just didn't want to change too much at once.

Reply:
The hotfix tags don't have an ARM build anyway. I've been using a more recent image locally for quite a while.